src/test/java/w3schools/forms/SelectMultipleForm.java

```diff
@@ -0,0 +1,93 @@
+package w3schools.forms;
+
+import aquality.selenium.browser.AqualityServices;
+import aquality.selenium.elements.interfaces.IButton;
+import aquality.selenium.elements.interfaces.IMultiChoiceComboBox;
+import aquality.selenium.elements.interfaces.ITextBox;
+import aquality.selenium.forms.Form;
+import org.openqa.selenium.By;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.function.IntSupplier;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class SelectMultipleForm extends Form {
+
+    private final IMultiChoiceComboBox cbxCars = getElementFactory().getMultiChoiceComboBox(By.id("cars"), "Cars");
+    private final IButton btnSubmit = getElementFactory().getButton(By.cssSelector("input[type='submit']"), "Submit");
+    private final ITextBox txbResult = getElementFactory().getTextBox(By.cssSelector(".w3-large"), "Result");
+
+    public SelectMultipleForm() {
+        super(By.id("iframe"), "Select Multiple");
+    }
+
+    public void switchToResultFrame() {
```

Comment:
I've noticed switchToResultFrame is always together with selectAll
won't be better to do this inside of selectAll?